quizapp/src/main/java/billennium/quizapp/resource/candidate/CandidateToSaveDto.java

```diff
@@ -0,0 +1,16 @@
+package billennium.quizapp.resource.candidate;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class CandidateToSaveDto {
```

Comment:
sadze ze to DTO moze byc niezmienne i finalne

Reply:
może wczęsniej za bardzo nie zwracałem na to uwagi ale jeżeli to możliwe warto by DTOsy były immutable. Wiesz dlaczego i w jakich przypadkach?

Reply:
podejrzewam ze sa latwiejsze w walidacji  np przy save zeby nie zmianic czasem czegos co by powodwalo wyjatek ?

Reply:
tak, przy zapisie lub odczycie chcemy aby w drodze np. pomiedzy odebraniem dto a zapisem nic nie zostało przypadkiem zmienione. Wtedy widać, że ktoś świadomie zrobił obiekt niezmienny i kuma o co chodzi. zrob ten dto w taki wlasnie sposob

Reply:
ta implementacja jest bezsensu, przemyśl to